### PR TITLE
Working on tasks in Dashbord

### PR DESCRIPTION
**Pull request by achrafbsibiss**

### CODE REVIEW

File: app/controllers/dashboard_controller.rb [hunk 1]
```diff
@@ -2,5 +2,6 @@ class DashboardController < ApplicationController
   def index
     @annoucements = Announcement.order(created_at: :desc).limit(3)
     @users = User.order(created_at: :desc).limit(5)
+    @tasks = current_user.task
```
Comment (itsAndL) [inline app/controllers/dashboard_controller.rb:5]: current_user.tasks

in the models/user.rb
has_many :tasks, dependent: :destroy
not
has_many :task, dependent: :destroy

File: app/controllers/dashboard_controller.rb [hunk 2]
```diff
@@ -2,5 +2,6 @@ class DashboardController < ApplicationController
   def index
     @annoucements = Announcement.order(created_at: :desc).limit(3)
     @users = User.order(created_at: :desc).limit(5)
+    @tasks = current_user.tasks
```
Comment (mohamedrez) [inline app/controllers/dashboard_controller.rb:5]: Only the ones that are active

**Review by mohamedrez — CHANGES_REQUESTED** [review]

**Comment by mohamedrez** [comment]
needs to fix tests, also not all the tasks the member can mark as complete so there is no need for the select box in the corner

File: app/controllers/dashboard_controller.rb (hunk 2 above)
Comment (achrafbsibiss) [reply app/controllers/dashboard_controller.rb:5]: can you explain it more

**Review by mohamedrez — APPROVED** [review]

**Pull request merged**